e2e-tests/insurance/cypress/e2e/journeys/your-business/turnover/turnover-page.spec.js

```diff
@@ -55,7 +56,7 @@ context('Insurance - Your business - Turnover page - As an Exporter I want to en
       cy.completeAndSubmitNatureOfYourBusiness();
 
       url = `${baseUrl}${ROOT}/${referenceNumber}${TURNOVER}`;
-      brokerUrl = `${baseUrl}${ROOT}/${referenceNumber}${BROKER}`;
+      creditControlUrl = `${baseUrl}${ROOT}/${referenceNumber}${CREDIT_CONTROL}`;
```

Comment:
this file is called turnover-page?

Reply:
Yes, the turnover page now redirects to the "credit control" route on submission, instead of "broker".